backend/src/api/group/content-types/group/schema.json

```diff
@@ -59,7 +59,16 @@
         "Community",
         "Education",
         "Activities"
```

Comment:
Can there be one more enum "Music"? before Activities

Reply:
I added it👍